postman/trackr.postman_collection.json

```diff
@@ -1,6 +1,6 @@
 {
```

Comment:
Should we create a set of API calls or an additional Postman collection with URLs for the DEV environment?

Reply:
We can add for iteration 3 but keep in mind that we have the UI now and folks are just going to use that. We don't have a use case integration with other application via API although that would be super cool.